plugin/external/@wordpress/elements/index.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
@aduth blocks has things like `getBlocks` so seems a different abstraction. As far as I can tell element is more like a wrapper for a component abstraction.